src/main/java/net/dv8tion/jda/internal/entities/RichPresenceImpl.java

```diff
@@ -181,6 +181,13 @@ public Party getParty()
         return party;
     }
 
+    @Nullable
+    @Override
+    public Emoji getEmoji()
```

Comment:
This override isn't needed, or is it? ActivityImpl already implements it exactly the same way + RichPresence just inherits it from there